backend/webapp/routes/__init__.py

```diff
@@ -236,7 +236,7 @@ def validation_tool_routes():
         methods=["POST"],
     )
     validation_tool.add_url_rule(
-        rule="/validation/payment_info",
+        rule="/validation/payment_info/<action>",
```

Comment:
Do not waste your time on changing it now, but in general it should be part of the data and not in the URL. The action is a property of the payment and not a part of the payment resource identifier.